pkg/dipper/encryption_test.go

```diff
@@ -109,3 +109,49 @@ data:
 	assert.Equal(t, "ENC[driver1,YWFiYmNjZGQ=]", MustGetMapDataStr(data, "data.item3.item4"), "item4 should be stripped off one deferred flag")
 	assert.Equal(t, "not encrypted", MustGetMapDataStr(data, "data.item1"), "data.item1 should remain unchanged")
 }
+
+func TestDecryptAllWithLookUp(t *testing.T) {
+	doc := `
+data:
+  item1: not encrypted
+  item2: LOOKUP[kvstore,foo]
```

Comment:
can we test something with the `ENC` prefix too? and testing around base64 encoded (or not) strings?

Reply:
There have been tests for `ENC[` in the same file from before.